.github/workflows/tests.yaml

```diff
@@ -12,18 +10,21 @@ jobs:
   spm:
     name: "iOS ${{ matrix.sdk }}"
 
-    runs-on: macos-11
+    runs-on: macos-12
 
     strategy:
       fail-fast: false # Don’t fail-fast so that we get all snapshot test changes
       matrix:
         include:
           - sdk: "14.5"
-            destination: platform=iOS Simulator,OS=14.5,name=iPhone 8
+            destination: platform=iOS Simulator,OS=14.5,name=iPad (8th generation)
```

Comment:
Installing iOS 14.5 with `xcodes` doesn't create an iPhone 8 sim for some reason, so I just picked another 2x device.